src/content/docs/tutorials/plot-data-with-charts.mdoc

```diff
@@ -0,0 +1,470 @@
+
+```tql
+from (
+  {time: 1, a: 10, b: 20},
+  {time: 2, a: 8, b: 25},
+  {time: 3, a: 14, b: 30},
+  {time: 4, a: 10, b: 25},
+  {time: 5, a: 18, b: 40},
+)
+chart_area x=time, y={a: a, b: b}
+```
+
+![Area chart](./plot-data-with-charts/chart-area.png)
+
+The area under the curve gives you a strong visual impression of the total
+event volume over time.
+
+#### Stack multiple series
+
+Like bar charts, area charts can display **stacked series**. This means that the
+values of the series add up, helping you *compare contributions* from different
+groups while still highlighting the overall cumulative shape.
+
+Pass `position="stacked" to see the difference`:
+
+```tql
+from (
+  {time: 1, a: 10, b: 20},
+  {time: 2, a: 8, b: 25},
+  {time: 3, a: 14, b: 30},
+  {time: 4, a: 10, b: 25},
+  {time: 5, a: 18, b: 40},
+)
+chart_area x=time, y={a: a, b: b}, position="stacked"
+```
+
+![Stacked area chart](./plot-data-with-charts/chart-area-stacked.png)
+
+Notice the difference in the y-axis interpretation:
+
+- Without stacking, the areas *overlap* each other.
+- With stacking, the areas become *disjoint* and *cumulatively add up* to the
+  total height.
+
+## Optimize Plotting with Inline Expressions
+
+While it's intuitive to *first* prepare your data and *then* start thinking
+about how to parameterize your chart operator, this leaves some opportunities
+for optimization and better results on the table. That's why the `chart_*`
+operators offer additional options for inline filtering, rounding, and
+summarization. These options allow you often to immediately jump to charting
+without having to think too much about data prepping.
+
+To appreciate these optimization, let's start with our metrics pipeline from the
+above line chart example:
+
+```tql
+metrics "process"
+where timestamp > now() - 1d                         // filtering
+set timestamp = timestamp.round(15min)               // rounding
+summarize timestamp, mem=mean(current_memory_usage)  // aggregation
+chart_line x=timestamp, y=mem
+```
+
+![Line chart with summarize](./plot-data-with-charts/chart-line-summarize.png)
+
+You can push the filtering, rounding, and aggregation into the chart operator:
+
+```tql
+metrics "process"
+chart_line (
+  x=timestamp,
+  y=mean(current_memory_usage), // aggregation
+  resolution=15min,             // rounding (= flooring)
+  x_min=now() - 1d,             // filtering
+)
+```
+
+![Line chart with resolution](./plot-data-with-charts/chart-line-resolution.png)
+
+Note how this make the pipeline more succinct by removing the extra `where`,
+`set`, and `summarize` operators.
+
+Here are some important details to remember:
+
+- The `x_min`/`x_max` (also `y_min`/`y_max`) options set the visible axis domain
+  to fixed interval. Use these options to crop or expand the viewport.
+- When using the `x_min` (also `x_max`) options, the chart operator implicitly
+  pushes up a `where` predicate, making it obsolete to restrict your charting
+  data via `where`.
```

Comment:
I think "pushes up" is too technical. Also, `where` is not a predicate, it's an operator. I suggest phrasing it like this:

```suggestion
- When using the `x_min` or `x_max` options, the chart operator implicitly
  filters your data for the correct time range, just as if you had specified
  `where x >= floor(x_min, resolution) and x < ceil(x_max, resolution)`.
  This avoids needing to specify the ranges twice, and it makes sure that the
  resolution is taken into account correctly.
```